### PR TITLE
Solutions for CSES Monsters and CSES Sliding Median

### PR DESCRIPTION
**Pull request by IsaacNoel**

### CODE REVIEW

**Comment by bqi343** [comment]
Didn't do step 3 of 

https://usaco.guide/general/contributing#adding-a-solution.

Also, the time complexity should be stated at the top in the same format as described by that module.

**Comment by IsaacNoel** [comment]
I've implemented those changes, lmk if I need to do anything else.

File: solutions/CSES Sliding Median.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+source: CSES
+title: Sliding Median
+author: Isaac Noel
+---
+
+**Time Complexity:** $\mathcal{O}(N\log N)$
+
+# Abstract
+
+Finding the median of a sliding window by maintaining two multisets in $N\log N$
```
Comment (bqi343) [inline solutions/CSES Sliding Median.mdx:12]: No need to restate the complexity here.

File: solutions/CSES Sliding Median.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+id: cses-1076
+source: CSES
+title: Sliding Median
+author: Isaac Noel
+---
+
+**Time Complexity:** $\mathcal{O}(N\log N)$
+
+# Abstract
```
Comment (bqi343) [inline solutions/CSES Sliding Median.mdx:10]: Use second-tier headings (## instead of #) to be consistent with how other solutions are formatted.

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Pull request merged**